tests/related_resource/api/resources.py

```diff
@@ -278,3 +280,24 @@ class Meta:
         queryset = Contact.objects.prefetch_related('groups')
         resource_name = 'contact'
         authorization = Authorization()
+
+
+class ConstantResource(ModelResource):
+    class Meta:
+        always_return_data = True
+        queryset = Constant.objects.all()
+        authenticaion = Authentication()
+        authorization = ReadOnlyAuthorization()
+
+
+class EquationResource(ModelResource):
+    description = fields.CharField()
+    constants = fields.ToManyField(
+        ConstantResource, 'constants', related_name='equations', null=True,
+        blank=True, full=True)
```

Comment:
If you set readonly=True here, the tests will pass. I think the 401 is expected behavior, however deleting the related items isn't, the error should be raised before any changes are made to the data.